packages/fluentui/react-northstar/src/components/Menu/menuContext.ts

```diff
@@ -0,0 +1,62 @@
+import { createContext } from '@fluentui/react-context-selector';
+import { ComponentVariablesInput } from '@fluentui/styles';
+import * as React from 'react';
+import { MenuItemProps } from './MenuItem';
+import { Accessibility } from '@fluentui/accessibility';
+
+export type MenuContextValue = {
+  activeIndex: number;
+  variables: ComponentVariablesInput;
+  pointing: boolean | 'start' | 'end';
+  primary: boolean;
+  underlined: boolean;
+  iconOnly: boolean;
+  vertical: boolean;
+  secondary: boolean;
+  pills: boolean;
+  inSubmenu: boolean;
+  onItemClick?: (e: React.KeyboardEvent | React.MouseEvent, itemIndex: MenuItemProps) => void;
+  accessibility: { [childBehaviorSlot: string]: Accessibility<any> };
+  __parentComponent: React.ElementType;
+};
+
+export type MenuContextSubscribedValue = Partial<
+  Pick<
+    MenuContextValue,
+    | 'activeIndex'
+    | 'variables'
+    | 'onItemClick'
+    | 'primary'
+    | 'pointing'
+    | 'underlined'
+    | 'iconOnly'
+    | 'vertical'
+    | 'inSubmenu'
+    | 'pills'
+    | 'secondary'
+    | 'accessibility'
+    | '__parentComponent'
```

Comment:
I would expect that we will have a separate context for this as I see we need under it at least: `menu`, `item`, `divider`

Reply:
Not sure if I get what you mean.. @mnajdova Maybe have a idea here if you are referring to the `__parentComponent`.

Reply:
Yeah, we can do different context, as it's needed only in Menu & MenuItem, not divider

Reply:
Ok

Reply:
Should I create 2 different context ( one for item and one for divider ) and use like 
 

```jsx
<MenuDividerContextProvider value={itemProps}>
     <MenuItemContextProvider value={itemProps}>
         {childrenExist(children) ? children : renderItems()}
     </MenuItemContextProvider>
</MenuDividerContextProvider>
```

 
Or you meant to split the Menu context and use something like:
 

```jsx
 <MenuContextProvider value={{ item: itemProps, divider: dividerProps }}>
     {childrenExist(children) ? children : renderItems()}
 </MenuContextProvider>
```

Reply:
Reverted